lib/webpack.config.js

```diff
@@ -1,18 +1,21 @@
+const path = require('path');
 const webpack = require('webpack');
 const CleanWebpackPlugin = require('clean-webpack-plugin');
 const UglifyJSPlugin = require('uglifyjs-webpack-plugin');
-const CopyWebpackPlugin = require('copy-webpack-plugin');
 
 const config = require('./config.json');
 
-var customSkin = typeof config.skin === 'object' ? config.skin : {};
-
 module.exports = {
     entry: {
         'cast' : './src/app/cast.js',
         'cast.min' : './src/app/cast.js',
     },
     devtool: 'source-map',
+    devServer: {
+        contentBase: path.join(__dirname, ''),
+        host: '10.66.172.59',
```

Comment:
Is this right? I think this is your ip. Shouldn't it be something like localhost?

Reply:
It can't be localhost, because sender (standalone OS in fact) will try to access it trough network by http. So it must be an IP in local network at least. But you're right, no reason to hardcode IP here, since it won't work for anybody beside me. Moved it into .env file, added .env into .gitignore, created README.md section about how to make it working.